src/pages/slash-graphql/admin/drop-data.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Drop Data'
+---
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+### Dropping complete data and retain schema
```

Comment:
Titles should match with the description. So something like this

### Dropping all Data Programatically

It is possible to drop all data while retaining your schema. This can be done from the `Settings` tab, by clicking the `Drop Data` button.

If you would like to do this programatically, this is possible by calling `dropData` on your instances admin endpoint (copy the bits about how to authenticate and find endpoint from the export documentation).

example curl here (just put any one... i prefer the application/graphql one since there are no variables).

If you would like to drop all data and also remove the schema, then you can use the following snippet.